packages/@coorpacademy-components/src/molecule/feedback/index.js

```diff
@@ -9,7 +12,7 @@ const Feedback = (props, context) => {
   const resource = media &&
     media.type && {
       ...media,
-      ...getOr({}, 'src.0', media)
+      ...pipe(get('src.0'), extend({description: mediaDescription}))(media)
```

Comment:
why do you need to change this logic whereas it's only to remove lives in popin-end header ?

Reply:
for a popinEnd with pdf ressource, the description of the resource was not displayed